Formula/uwsgi.rb

```diff
@@ -13,6 +13,13 @@ class Uwsgi < Formula
     sha256 "9cc48b037ae97dab7c026a0c6af3021f984de4446d0e8ac089f38c62a06a97ff" => :mavericks
   end
 
+  # Fails when SDKROOT is set: "no such file or directory: '... libpython2.7.a'"
+  # Reported 23 Jun 2016: https://github.com/unbit/uwsgi/issues/1299
+  patch do
+    url "https://raw.githubusercontent.com/ilovezfs/formula-patches/2a64b90f/uwsgi/libpython-tbd-xcode-sdk.diff"
```

Comment:
Did we not submit this one to the Homebrew repo? You can push to that repo directly rather than filing PRs, FYI.